api.py

```diff
@@ -367,7 +389,130 @@ def set_generic_data(dataset_id, key, data_string, **kwargs):
     return ingest_entities(dataset_id, {'entities': [entity]}, **kwargs)
 
 
+def convert_coordinates(point):
+    if len(point) > 2:
+        raise KeyError('A point should only have two dimensions', point)
```

Comment:
I used `KeyError` a lot.  Basically, any time the data structure is not in compliance with the documentation.

`KeyError` is documented as:
>Raised when a mapping (dictionary) key is not found in the set of existing keys.